src/axolotl/utils/schemas/validation.py

```
@@ -815,22 +815,22 @@ def check_fsdp_version(cls, data):
                 "For more details on migrating your config. "
             )
         return data
+    
+    @model_validator(mode="before")
+    @classmethod
+    def check_fsdp2_cpu_offload_pin_memory(cls, data):
+        if not (fsdp_config := data.get("fsdp_config")): return data
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)